plugins/woocommerce/tests/php/includes/emails/class-wc-email-customer-pos-refunded-order-test.php

```
@@ -422,4 +422,59 @@ public function test_email_content_does_not_include_refund_returns_policy_when_o
+		$order->set_status( 'completed' );
+		$order->save();
+
+		// Create two refunds so the order has multiple.
+		$first_refund = wc_create_refund(
+			array(
+				'order_id' => $order->get_id(),
+				'amount'   => '5.00',
+				'reason'   => 'First refund',
+			)
+		);
+		$second_refund = wc_create_refund(
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$second_refund'. (undefined)

(UnusedLocalVariable)